faq/SUMMARY.md

```diff
@@ -4,41 +4,41 @@
 
 ## 📙 FAQ
 
-* [🦊 ShapeShift](faq/about-shapeshift/README.md)
-  * [🔮 The ShapeShift DAO](faq/about-shapeshift/the-shapeshift-dao.md)
-  * [🎨 ShapeShift Branding](faq/about-shapeshift/shapeshift-branding.md)
-  * [🔄 Contribute](faq/about-shapeshift/contribute/README.md)
-    * [💰 Bounties](faq/about-shapeshift/contribute/bounties.md)
-    * [💻 Codebase](faq/about-shapeshift/contribute/codebase.md)
-  * [🔸 What Is A DEX?](faq/about-shapeshift/what-is-a-dex.md)
-  * [🎗 Important Links When Interacting with the ShapeShift DAO](faq/about-shapeshift/important-links-when-interacting-with-the-shapeshift-dao.md)
-  * [📡 FOX Current CEX/DEX Listings](faq/about-shapeshift/daos-current-cex-dex-listings.md)
-  * [🔤 What are Your Terms of Service?](faq/about-shapeshift/what-are-your-terms-of-service.md)
-  * [🗳 Governance](faq/about-shapeshift/governance.md)
-* [🌐 App.ShapeShift](faq/app.shapeshift/README.md)
-  * [🚀 What is app.shapeshift.com?](faq/app.shapeshift/what-is-app.shapeshift.com.md)
-  * [🛠 Open Source](faq/app.shapeshift/open-source.md)
-  * [⛓ Blockchains](faq/app.shapeshift/blockchains.md)
-  * [🪜 Supported ON/OFF Ramps](faq/app.shapeshift/supported-on-off-ramps.md)
-* [🎛 Protocols](faq/protocols/README.md)
-  * [💰 DeFi Section - Overview](faq/protocols/defi-section-overview.md)
-  * [🚜 Liquidity Pools & Farming](faq/protocols/liquidity-pools-and-farming.md)
-  * [⛱ Yieldies](faq/protocols/yieldies.md)
-  * [🔨 THORSwap](faq/protocols/thorswap.md)
-  * [🐮 CoW Swap](faq/protocols/cow-swap.md)
-  * [🌀 0x](faq/protocols/0x.md)
-  * [🔓 Yearn](faq/protocols/yearn.md)
-  * [🌌 Cosmos](faq/protocols/cosmos.md)
-  * [🦠 Osmosis](faq/protocols/osmosis.md)
-  * [🛡 Idle Finance](faq/protocols/idle-finance.md)
-  * [🎁 Tokemak](faq/protocols/tokemak.md)
-  * [🪙 Validators](faq/protocols/validators.md)
-* [👛 Wallets](faq/wallets/README.md)
-  * [🔐 KeepKey](faq/wallets/keepkey/README.md)
-    * [⚒ KeepKey Set Up](faq/wallets/keepkey/keepkey-set-up.md)
-    * [🎛 KeepKey Settings](faq/wallets/keepkey/keepkey-settings.md)
-    * [🪡 Troubleshooting KeepKey](faq/wallets/keepkey/troubleshooting-keepkey.md)
-    * [🏥 How Do I Recover On My KeepKey?](faq/wallets/keepkey/how-do-i-recover-on-my-keepkey.md)
-    * [♨ KeepKey Privacy Policy](faq/wallets/keepkey/keepkey-privacy-policy.md)
-    * [📜 KeepKey Terms of Service](faq/wallets/keepkey/keepkey-terms-of-service.md)
-* [📱 Mobile](faq/mobile.md)
+* [🦊 ShapeShift](about-shapeshift/README.md)
```

Comment:
`.gitbook.yaml` defines the summary as `./faq.*` - everything else is now relative to that